cbv/management/commands/populate_cbv.py

```diff
@@ -4,14 +4,46 @@
 
 import django
 from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
 from django.core.management.base import BaseCommand
+from django.utils.functional import Promise
 
 from blessings import Terminal
 from cbv.models import Project, ProjectVersion, Module, Klass, Inheritance, KlassAttribute, ModuleAttribute, Method, Function
 
 t = Terminal()
 
 
+class LazyAttribute(object):
+    functions = {
+        'gettext': '_',
```

Comment:
I wonder if it would be better to show the resulting function as `gettext_lazy` rather than `_` (same for ugettext). Its less ambiguous.